tests/restmodel/test_container.py

```diff
@@ -131,7 +131,7 @@ def test_retrieve(self):
             self.assertEqual(res.code, httpstatus.NOT_FOUND)
 
             self._app.container_manager.docker_client.containers = \
-                mock_coro_factory(return_value=[utils.containers_dict()])
+                mock_coro_factory(return_value=[containers_dict()])
```

Comment:
I wonder if we can mock the internal sync client instead...

Just a comment. I would not do it now, but useful to consider. I changed the AsyncDockerClient.client to _sync_client, so if you change it now it would need change again. Let's wait.